src/api/wix/WixToolset.Data/WixOutput.cs

```diff
@@ -60,6 +60,24 @@ public static WixOutput Create(string path)
             return WixOutput.Create(uri, stream);
         }
 
+        /// <summary>
+        /// Creates a new file structure on disk that can only be written to.
+        /// </summary>
+        /// <param name="path">Path to write file structure to.</param>
+        /// <returns>Newly created <c>WixOutput</c>.</returns>
+        public static WixOutput CreateNew(string path)
```

Comment:
For the sake of minimal change and given we want to improve this API, let's hide this for now and make it `internal`.